multipicker/src/main/java/im/vector/lib/multipicker/ContactPicker.kt

```diff
@@ -72,10 +73,13 @@ class ContactPicker : Picker<MultiPickerContactType>() {
                                 selection,
                                 selectionArgs,
                                 null
-                        )?.use { cursor ->
-                            while (cursor.moveToNext()) {
-                                val mimeType = cursor.getString(cursor.getColumnIndex(ContactsContract.Data.MIMETYPE))
-                                val contactData = cursor.getString(cursor.getColumnIndex(ContactsContract.Data.DATA1))
+                        )?.use inner@{ innerCursor ->
```

Comment:
this is my first time seeing `inner@{`  what does it do?

Reply:
"inner" is just a label, it could be anything else. It allows to use `return@inner` below (lines 77 and 78). Using classical `return@use` is not possible there since we are in 2 nested `use` blocks, so is ambiguous.

Reply:
ah thanks for explaining